collector/btrfs_linux.go

```diff
@@ -275,6 +275,30 @@ func (c *btrfsCollector) getMetrics(s *btrfs.Stats, ioctlStats *btrfsIoctlFsStat
 			metricType: prometheus.GaugeValue,
 			value:      float64(s.Allocation.GlobalRsvSize),
 		},
+		{
+			name:       "commits_total",
+			desc:       "The total number of commits that have occurred.",
+			metricType: prometheus.CounterValue,
+			value:      float64(s.CommitStats.Commits),
+		},
+		{
+			name:       "last_commit_seconds",
```

Comment:
Wondering if we even need last/max, given we can calculate them from total

Reply:
@SuperQ wdyt?

Reply:
According to [the docs](https://btrfs.readthedocs.io/en/latest/btrfs-man5.html#sysfs-interface) the reported value for the max can be (re)set by writing into the file the statistics are read from (`commit_stats`), so if that is used I think the value can't be calculated anymore.